email_templates/carpeEmail/style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: red;
+}
```

Comment:
Oh yeah! Got it. Did y'all run into the issue of not having inline styling, or does handlebars compile down into something that sends correctly?